blueman/plugins/applet/TransferService.py

```diff
@@ -66,6 +66,7 @@ def on_action(action):
             logging.info("Action %s" % action)
 
             if action == "accept":
+                assert self._pending_transfer
```

Comment:
Why, self._pending_transfer is created in the parent function before creating the notification.

Reply:
I give you that one. That's rather obvious from the code as `on_action` does not get referenced before `_pending_transfer` gets set but mypy does not know. There's even one small exception though: `__init__` might get called on the already initialized object so that `_pending_transfer` would get reset to `None`.